index.html

```diff
@@ -5,9 +5,31 @@
     <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
     <title>Matcha</title>
     <link rel="stylesheet" href="css/styles.css">
-  </head>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap-theme.min.css">
+	</head>
+
   <body id='body'>
-    <div id="App"></div>
+    <div id="App">
+		<!-- c'est ici qu'il y aura tous le react -->
+	 </div>
     <script type="text/javascript" src='js/bundle.js'></script>
   </body>
+
+
+
+	<!--[if lt IE 9]>
```

Comment:
are you sure, we need this ??
i thought we needed just to run it in firefox and chrome